samples/subsys/shell/shell_module/README.rst

```diff
@@ -0,0 +1,115 @@
+.. zephyr:code-sample:: shell-module
+   :name: Custom Shell module
+   :relevant-api: shell_api
+
+   Register shell commands using the Shell API
+
+Overview
+********
+
+This is a simple application demonstrating how to write and register commands
+using the :ref:`Shell API <shell_api>`:
+
+Register Static commands
+   ```version``` is a static command that prints the kernel version.
```

Comment:
double backquotes, not triple, please

![image](https://github.com/zephyrproject-rtos/zephyr/assets/128251/b3bfa0c9-80bd-4968-8e27-04b5c1eae7b7)

Reply:
Roger - got confused by this suggestion: https://github.com/zephyrproject-rtos/zephyr/pull/71904#discussion_r1603638630.

Addressed.

Reply:
> Roger - got confused by this suggestion: [#71904 (comment)](https://github.com/zephyrproject-rtos/zephyr/pull/71904#discussion_r1603638630).

woops, sorry!